### ISSUE TITLE
IOERROR invalid directory

### ISSUE THREAD

**Issue by ccharoux** [kivy/buildozer#599]
**Trying to buid one apk
Nearly finished** 
[DEBUG]:   	
[DEBUG]:   	BUILD SUCCESSFUL
[DEBUG]:   	Total time: 19 seconds
[INFO]:    <- directory context /home/kivy/.buildozer/android/platform/python-for-android-master
[INFO]:    # Copying APK to current directory
[INFO]:    # Found APK file: /home/kivy/.buildozer/android/platform/build/dists/Approche/bin/Approche-0.1-debug.apk
[DEBUG]:   -> running cp /home/kivy/.buildozer/android/platform/build/dists/Approche/bin/Approche-0.1-debug.apk ./
WARNING: Received a --sdk argument, but this argument is deprecated and does nothing.
Traceback (most recent call last):
  File "/usr/local/bin/buildozer", line 11, in <module>
    load_entry_point('buildozer==0.33', 'console_scripts', 'buildozer')()
  File "/usr/local/lib/python2.7/dist-packages/buildozer/scripts/client.py", line 13, in main
    Buildozer().run_command(sys.argv[1:])
  File "/usr/local/lib/python2.7/dist-packages/buildozer/__init__.py", line 1058, in run_command
    self.target.run_commands(args)
  File "/usr/local/lib/python2.7/dist-packages/buildozer/target.py", line 91, in run_commands
    func(args)
  File "/usr/local/lib/python2.7/dist-packages/buildozer/target.py", line 103, in cmd_debug
    self.buildozer.build()
  File "/usr/local/lib/python2.7/dist-packages/buildozer/__init__.py", line 212, in build
    self.target.build_package()
  File "/usr/local/lib/python2.7/dist-packages/buildozer/targets/android.py", line 808, in build_package
    copyfile(join(apk_dir, apk), join(self.buildozer.bin_dir, apk_dest))
  File "/usr/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
**IOError: [Errno 2] No such file or directory: u'/home/kivy/.buildozer/android/platform/build/dists/Approche/build/outputs/apk/Approche-debug.apk'**

**Comment by inclement** [comment]
Thanks for the report. This should be fixed by https://github.com/kivy/buildozer/pull/597, closing this as a duplicate of that.

**Comment by NimrodSegall** [comment]
I am having the same issue on the newest version. Was this resolved?

**Comment by AndreMiras** [comment]
Yes and no apparently. It solved it for some setup, but not for other.
Apparently the detection method is not reliable enough, see comment https://github.com/kivy/buildozer/pull/597#issuecomment-352186895
So until this is fixed and because I don't have time to investigate, I implemented an ugly workaround on buildozer==0.34.
Disclaimer don't try it at home...
I changed `if is_gradle_build:` by `if not is_gradle_build:` this is stupid as hell but I didn't have much time to investigate this one for now.
